<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForcesWidget</class>
 <widget class="QWidget" name="ForcesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forcing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tw_Forcing"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
